pkg/client/clientset.go

```diff
@@ -255,13 +248,13 @@ func validateURL(dtURL string) error {
 	return nil
 }
 
-func CreateClientSet(url string, auth manifest.Auth, opts ClientOptions) (*ClientSet, error) {
+func CreateClientSet(ctx context.Context, url string, auth manifest.Auth, opts ClientOptions) (*ClientSet, error) {
 	var (
 		classicClient, client *corerest.Client
```

Comment:
No, it is OK for the current design because either they are set to non-nil values or this function errors out:

https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/bad912d17a3c594ee78b5196194aab774b4e79ce/pkg/client/clientset.go#L325-L337

We could certainly improve the clientset creation though